README.md

```diff
@@ -26,33 +26,41 @@ const config = {
     port: 21,
     localRoot: __dirname + "/local-folder",
     remoteRoot: "/public_html/remote-folder/",
-    // include: ["*", "**/*"],      // this would upload everything except dot files
-    include: ["*.php", "dist/*", ".*"],
+    // typical wordpress includes
```

Comment:
You could use JSDoc here with the example together. This way it is possible to get an IDE hint for `include`.